src/main/java/org/apache/maven/buildcache/checksum/MavenProjectInput.java

```diff
@@ -819,4 +833,22 @@ public int compare( Path f1, Path f2 )
         }
     }
 
+    public static boolean shouldSkipCacheLookup( MavenProject project )
```

Comment:
`skipCacheLookup` or `isSkipCacheLookup` will be better

Reply:
@AlexanderAshitkin fixed